src/helpers.ts

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Try not to disable the linter, but to describe everything that is needed so that it does not quarrel